http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -53,7 +59,7 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport)(
                 }
               }
             }
-          } ~
+          } ~ path("jupyter" | "rstudio") {
```

Comment:
this doesn't actually work. How its designed, /jupyter/x will not get picked up by anything. I need to add another block after this to capture `jupyter/*`

Reply:
`pathPrefix` might help here